packages/components/molecules/f-user-message/src/components/UserMessage.vue

```diff
@@ -2,7 +2,10 @@
     <div
         v-if="userMessageEnabled"
         :data-theme="theme"
-        :class="$style['c-userMessage']"
+        :class="[
```

Comment:
I think this works too (saving the use of a ternary):

```javascript
:class="{
      [$style['c-userMessage']]: true,
      [$style['c-userMessage--flush']]: isFlush
]}"

Reply:
Agree, also that's how the [Vue docs](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax) explain how to add conditional classes, so good to stick with standard syntax I reckon 😺 

You can actually just specify the single class as conditional.

:class="[$style['c-userMessage'], {
    [$style['c-userMessage--flush']]: isFlush
}]"

Reply:
Applied suggestion cheers